### ISSUE TITLE
BoilR breaks Steam Big Picture

### ISSUE THREAD

**Issue by ErikApption** [PhilipK/BoilR#384]
**To Reproduce**
1. Sync Xbox, Epic, UPlay to steam with boilr
2. Open Steam 
3. Launch big picture
4. Big picture is empty

**Expected behavior**
Big picture shows all the games

**Screenshots**
Big Picture
![image](https://github.com/PhilipK/BoilR/assets/3179656/7f37df10-1a29-4e0a-be5a-2f823b14949d)

Regular mode
![image](https://github.com/PhilipK/BoilR/assets/3179656/811a50d6-36fb-4fa8-bd99-4669a3b8c32d)

**Desktop (please complete the following information):**
 - OS: win 11
 - BoilR Version 1.9.4

**Comment by pillepalle123** [comment]
Yes, I have exactly the same issue. It is not always empty though. In the beginning it seemed fine in big picture mode but then this happened. Removing all non steam game fixed it. Also deinstalling steam helped. Maybe it is a certain game? Or it is renaming a game after import directly within steam which I did....
It happen again multiple times after that. Thx for the app and support!

**Comment by PhilipK** [comment]
Have you verified if this is because of the images or just the shortcuts? Could you try to import without the images? Is bigpicture still broken?
Maybe a big picture does not like a certain format of images?

**Comment by pillepalle123** [comment]
I have trouble reproducing it. It works for now.
If I remember correctly, i removed those dummy icons only, so without images downloaded. I think I hadn't had that option to automatically close and reopen Steam activated in the beginning, maybe that was part of the issue... I have an eye on it in case it happens again.

**Comment by ankur-p** [comment]
I had the same issue but was able to fix it by forcing the games to launch through Epic Launcher. Tried with and without images before trying this which does work with images. I haven't tried going through each individual game yet to see if only one or a few are causing the issue. I checked every game and tried launching a couple, and the ones I tested seem to be working fine. That's about the extent of my troubleshooting so far, but I thought I should let you know in case that helps uncover any potential underlying issue.
